lib/checks/navigation/region-after.js

```diff
@@ -0,0 +1,73 @@
+import cache from '../../core/base/cache';
+import { getNodeFromTree } from '../../core/utils';
+
+function regionAfter(results) {
+	const regionlessNodes = cache.get('regionlessNodes');
```

Comment:
I don't think we should use the cache as a way to pass information between the evaluate and after functions. That seems problematic. If we need info about a node, lets put it in `data`.